### ISSUE TITLE
Narrator announces drop-down item information twice in ComboBox Input text field

### ISSUE THREAD

**Issue by Sharvani-Ashok** [microsoft/fluentui#13772]
### Environment Information

- **Browser and OS versions**: Version 83.0.478.50 (Official build) (64-bit)

### Describe the issue:

When we use Narrator and go to the Input text field of the ComboBox dropdown through the down arrow key, the control information of the item is read twice (Ex: Basic combobox Option C combo init Option C)
The link Combobox:
https://developer.microsoft.com/en-us/fluentui#/controls/web/combobox

#### Actual behavior:

The narrator announces the control information of the item twice

#### Expected behavior:

Narrator should announce the control information of the item only once
